java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -984,6 +984,34 @@ public void hset_hget_existing_fields_non_existing_fields(BaseClient client) {
         assertNull(client.hget(key, "non_existing_field").get());
     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void non_UTF8_GlideString_test(BaseClient client) {
+        byte[] nonUTF8Bytes = new byte[] {(byte) 0xEE};
+        GlideString key = gs(nonUTF8Bytes);
+        GlideString hashKey = gs(UUID.randomUUID().toString());
+        GlideString hashNonUTF8Key = gs(new byte[] {(byte) 0xFF});
+        GlideString value = gs(nonUTF8Bytes);
+        String stringField = "field";
+        Map<GlideString, GlideString> fieldValueMap = Map.of(gs(stringField), value);
+
+        // Non UTF-8 key and value
```

Comment:
you should describe what you're testing. 
ie. testing keys and values using byte[] that cannot be converted to UTF-8 Strings.